### PR TITLE
Updated Simple CRUD apps to use new serverless connection UX

### PR DESCRIPTION
**Pull request by ericharmeling**
Related to https://github.com/cockroachdb/docs/pull/13525.
Fixes https://cockroachlabs.atlassian.net/browse/DOC-2040.

This PR updates the simple CRUD tutorials to align with the new connection UX updates. The PR also includes some other updates that help standardize our connection experience across the Console, application tutorials, and [connection reference docs](https://www.cockroachlabs.com/docs/stable/connect-to-the-database.html) (e.g., the use of the `DATABASE_URL` environment variable).

Note that I have made updates to all affected sample apps, to be consistent with the changes made in this PR.

(Also note that this PR actually includes changes from #13525. After that PR is merged into `master`, I will rebase this branch on `master` and force-push.)

After this PR is approved, I will copy the changes over to all other affected docset versions.

### CODE REVIEW

**Review by emily-horing — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13611)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling and @gemma-shay)

___
*[_includes/cockroachcloud/quickstart/create-first-sql-user.md, line 8 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hsHRGAHZ8ZMmkBkv1:-N-hsHRGAHZ8ZMmkBkv2:b-4ot4g0) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/create-first-sql-user.md#L8)):*
> ```Markdown
> 1. Click **Next**.
> 
>     Currently, all new users are created with full privileges. For more information and to change the default settings, see [Granting privileges](https://www.cockroachlabs.com/docs/cockroachcloud/user-authorization.html#grant-privileges) and [Using roles](https://www.cockroachlabs.com/docs/cockroachcloud/user-authorization.html#use-roles).
> ```

This section doesn't show up in the preview docs, should it?
___
*[_includes/cockroachcloud/quickstart/get-connection-string.md, line 6 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hpe8R3IzJs52DWRkU:-N-hpe8R3IzJs52DWRkV:bfc61mp) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/get-connection-string.md#L6)):*
> ```Markdown
> 1. Open the **General connection string** section, then copy the connection string provided and save it in a secure location.
> 
>     This Quickstart uses default certificates, so you can skip the **Download CA Cert** instructions.
> ```

Isn't this a CRUD app and not a quickstart?
___
*[_includes/v22.1/app/init-bank-sample.md, line 1 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hptWZ3Jn-QeCI8okh:-N-hptWZ3Jn-QeCI8oki:b28iouu) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/init-bank-sample.md#L1)):*
> ```Markdown
> 1. Set the `DATABASE_URL` environment variable to the connection string to your cluster:
> ```

nit: "for your cluster"
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 21 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hqN_-86OMAeVGD6eI:-N-hqN_-86OMAeVGD6eJ:b-n0vvkv) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L21)):*
> ```Markdown
> The **Connect to your cluster** dialog shows information about how to connect to your cluster.
> 
> 1. Select **Parameters** from the **Select option/language** dropdown.
> ```

i think i noted this on the quickstart but it probably will make sense for us to remove the /language part for now? We should update the UI too
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hrGl2C7iY9jD8TZo0:-N-hrGl2C7iY9jD8TZo1:b1ma653) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L22)):*
> ```Markdown
> 
> 1. Select **Parameters** from the **Select option/language** dropdown.
> 1. Copy the connection information for each parameter displayed and save it in a secure location.
> ```

This is just for my education: what do we actually mean when we say "secure location"? what would a secure location be?

**Comment by emily-horing** [comment]
Seems like the preview deploy doesn't match what I'm seeing in the change log. This seems off
Screen Shot 2022-04-15 at 11 41 26 AM

File: v22.1/build-a-typescript-app-with-cockroachdb.md [hunk 1]
```diff
@@ -84,6 +86,11 @@ This tutorial shows you how run a simple application built with [TypeORM](https:
```
Comment (gemma-shay) [inline v22.1/build-a-typescript-app-with-cockroachdb.md:86]: the routing id param looks like it's just called "Options" in the console. might be good to clarify that... or change the label in the console.
Screen Shot 2022-04-15 at 12 02 33 PM
.

File: v22.1/build-a-go-app-with-cockroachdb-pq.md [hunk 2]
```diff
@@ -16,109 +16,53 @@ This tutorial shows you how build a simple Go application with CockroachDB and t
...
-
-{% include_cached copy-clipboard.html %}
-~~~ go
-{% remote_include https://raw.githubusercontent.com/cockroachlabs/hello-world-go-pq/master/main.go %}
-~~~
-
-### Update the connection parameters
-
-<section class="filter-content" markdown="1" data-scope="local">
+1. Initialize the module:
```
Comment (gemma-shay) [inline v22.1/build-a-go-app-with-cockroachdb-pq.md:41]: have to `cd` into the directory before this

**Review by ericharmeling — COMMENTED** [review]
@emily-horing 

> Seems like the preview deploy doesn't match what I'm seeing in the change log. 

You might be looking at the wrong version. I've only made the changes to 22.1 files, so they appear under the `dev` version of the docs preview, and not the `stable` version. Sorry that's so confusing. 

For example, this URL will point to the node-postgres tutorial that includes the right "Start CockroachDB" setup steps:

https://deploy-preview-13611--cockroachdb-docs.netlify.app/docs/dev/build-a-nodejs-app-with-cockroachdb.html 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13611#-:-N-iIHSTEwSTe25l-LT-:balesiw)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing and @gemma-shay)

___
*[v22.1/build-a-go-app-with-cockroachdb-pq.md, line 41 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0UU-r1-41:-N-x2BeHEqvuspc394nZ:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/v22.1/build-a-go-app-with-cockroachdb-pq.md#L41)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

have to `cd` into the directory before this (same for gorm)
</blockquote></details>

Done.
___
*[v22.1/build-a-python-app-with-cockroachdb.md, line 88 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0Ud-r1-88:-N-x2LcA8fYwbXuk5UBC:brad2wp) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/v22.1/build-a-python-app-with-cockroachdb.md#L88)):*
>I think this might be missing a step where you cd into the hello-world-python-psycopg2. 

Done.

> I'm also getting the error
usage: example.py [-h] [-v] dsn example.py: error: the following arguments are required: dsn
when I try to run the code.

Sorry about that. I hadn't merged the changes to the sample app, as they were still in review by the driver maintainer. Just merged: https://github.com/cockroachlabs/hello-world-python-psycopg2/pull/7.
___
*[v22.1/build-a-python-app-with-cockroachdb-sqlalchemy.md, line 96 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0Uc-r1-96:-N-x3JlN4MsMW2_GNWSn:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/v22.1/build-a-python-app-with-cockroachdb-sqlalchemy.md#L96)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

Had to run `pip install virtualenv` first for Step 3
</blockquote></details>

Done.
___
*[v22.1/build-a-typescript-app-with-cockroachdb.md, line 86 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0Ue-r1-86:-N-x8TjA9bJrqmLAPjz7:b9gyz3u) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/v22.1/build-a-typescript-app-with-cockroachdb.md#L86)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

the routing id param looks like it's just called "Options" in the console. might be good to clarify that... or change the label in the console.
Screen Shot 2022-04-15 at 12 02 33 PM
.
</blockquote></details>

I updated the wording here a bit to be clearer. `--cluster=<routing-id>` is technically the value provided for the `options` parameter, so this console is correct. 
___
*[_includes/cockroachcloud/quickstart/get-connection-string.md, line 6 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0UN-r1-6:-N-iJ4XqB7Ah9mXuKyBA:bjkb0n0) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/get-connection-string.md#L6)):*
> make a new include for the sample apps

Let's do that, so I'm not stepping on your toes with app tutorial includes.

> Also, you still need to download the Cert for the python sample apps, right?

I'll use a different file for Python and Ruby apps.
___
*[_includes/cockroachcloud/quickstart/get-connection-string.md, line 6 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hpe8R3IzJs52DWRkU:-N-yItvD2XCPOUIExR4M:b-4kh18n) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/get-connection-string.md#L6)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

Isn't this a CRUD app and not a quickstart?
</blockquote></details>

See my comment below re: making new includes.
___
*[_includes/cockroachcloud/quickstart/get-connection-string.md, line 9 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0UN-r1-9:-N-iJsyi3Fb8bpQuqlqL:bad15rb) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/get-connection-string.md#L9)):*
> [**SQL Users** page](user-authorization.html).

Using relative references to files in the `cockroachcloud` directory breaks the build because the sample app tutorials are all under version directories (e.g., `v22.1`). Won't be an issue, as I'm going to just make new includes for the sample apps.
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 19 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0UQ-r1-19:-N-iKhrQ6LDg3QpOx_6-:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L19)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

```suggestion
The **Connect to cluster** dialog shows information about how to connect to your cluster.
```
</blockquote></details>

Done.
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 21 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-cnF45GrSHs1YdN0UQ-r1-21:-N-iKmrKAl1wsA-TVsfb:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L21)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

```suggestion
1. Select **Parameters only** from the **Select option** dropdown.
```
</blockquote></details>

Done.
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 21 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hqN_-86OMAeVGD6eI:-N-iKnMS4EZwgDc5-YPL:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L21)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

i think i noted this on the quickstart but it probably will make sense for us to remove the /language part for now? We should update the UI too
</blockquote></details>

Done.
___
*[_includes/v22.1/app/sample-setup-parameters.md, line 22 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hrGl2C7iY9jD8TZo0:-N-iKqkuDKR1dC95StfM:b-41jj75) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/sample-setup-parameters.md#L22)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

This is just for my education: what do we actually mean when we say "secure location"? what would a secure location be?
</blockquote></details>

Probably a secrets manager of some sort (e.g., aws secrets manager), or a password file (e.g., https://www.postgresql.org/docs/current/libpq-pgpass.html).
___
*[_includes/cockroachcloud/quickstart/create-first-sql-user.md, line 8 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hsHRGAHZ8ZMmkBkv1:-N-iJ0E8BH-D6uQxOQn4:bnd45rg) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/cockroachcloud/quickstart/create-first-sql-user.md#L8)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

This section doesn't show up in the preview docs, should it?
</blockquote></details>

See my comment above. 
___
*[_includes/v22.1/app/init-bank-sample.md, line 1 at r1](https://reviewable.io/reviews/cockroachdb/docs/13611#-N-hptWZ3Jn-QeCI8okh:-N-iKSmZERlT9LfSzOvv:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/78c88577524a68a3f7ebeb1a0b65501ba82062b5/_includes/v22.1/app/init-bank-sample.md#L1)):*
<details><summary><i>Previously, emily-horing wrote…</i></summary><blockquote>

nit: "for your cluster"
</blockquote></details>

Done.

**Review by ericharmeling — COMMENTED** [review]
@emily-horing @gemma-shay 

Thanks for the thorough and helpful reviews! I think I got to all of your comments, so this should be good to go now. I also added the updated Ruby tutorials, given that we got a few more days to work through the docs. 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13611#-:-N-yJEYaC-YDhWScTKPR:b-vedil1)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing and @gemma-shay)

File: v22.1/build-a-ruby-app-with-cockroachdb.md [hunk 3]
```diff
@@ -33,86 +29,82 @@ git clone https://github.com/cockroachlabs/hello-world-ruby-pg
...
-{% include_cached copy-clipboard.html %}
-~~~ shell
-ruby main.rb
-~~~
+1. Run the code to create a table and insert some rows:
 
-The output should be:
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    ruby main.rb
```
Comment (gemma-shay) [inline v22.1/build-a-ruby-app-with-cockroachdb.md:91]: got error:
```
main.rb:8:in `async_exec': ERROR:  value type int doesn't match type uuid of column "id" (PG::DatatypeMismatch)
HINT:  you will need to rewrite or cast the expression
```

File: v22.1/build-a-java-app-with-cockroachdb-hibernate.md [hunk 4]
```diff
@@ -24,17 +24,26 @@ For another use of Hibernate with CockroachDB, see our [`examples-orms`](https:/
...
 
-{% include {{page.version.version}}/app/create-a-database.md %}
+Clone the `example-app-java-hibernate` repo to your machine:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+git clone https://github.com/cockroachlabs/example-app-java-hibernate/
+~~~
 
-## Step 3. Run the Java code
```
[the remark this answers is not in the document]
Comment (ericharmeling) [reply v22.1/build-a-java-app-with-cockroachdb-hibernate.md:33]: Done.

**Review by gemma-shay — APPROVED** [review]

**Pull request merged**